### PR TITLE
Changed name of Federal District

### LINKED ISSUE #293: WOF doesn't have an entry for Mexico City
e.g.
https://whosonfirst.mapzen.com/spelunker/search/?q=mexico+city#4/31.04/-103.34

even Ciudad de México doesn't work but it finds a bunch of places around it
https://whosonfirst.mapzen.com/spelunker/search/?q=Ciudad+de+M%C3%A9xico#5/25.640/-106.500

is it supposed to be more-or-less this? https://whosonfirst.mapzen.com/spelunker/id/85686515/#9/19.4252/-99.1159

### PR DESCRIPTION
**Pull request by stepps00**
FIxes #293. Changed `wof:name` from Distrito Federal to Ciudad de Mexico. No geometry change. May need to change the county records that are parented by Ciudad de Mexico to `localadmin`, as the government restructured the capital government earlier this year.

### CODE REVIEW

**Comment by nvkelso** [comment]
👍 

Works. Alternatively you could have matched [Washington, DC](https://whosonfirst.mapzen.com/spelunker/id/85931779/#11/38.8937/-77.0146) to **create a new locality record**. But I like unitary authorities, so let's go with your PR. 

**Please file a new issue for:** Agree the [33 existing localities](https://whosonfirst.mapzen.com/spelunker/id/85686515/descendants/?&placetype=locality#10/19.3192/-99.1159) should be turned into a type of neighbourhood. Looks like many might be `borough` and most in the south-east might be `neighbourhood`. Then assign to Aaron to merge this one.

See also:

- https://en.wikipedia.org/wiki/Mexico_City
- https://en.wikipedia.org/wiki/Municipalities_of_Mexico_City
- https://en.wikipedia.org/wiki/List_of_neighborhoods_in_Mexico_City

**Comment by stepps00** [comment]
Filed [new issue](https://github.com/whosonfirst/whosonfirst-data/issues/297) for placetype issue in Mexico City. Sending this PR to Aaron for a merge to master.

File: data/856/865/15/85686515.geojson [hunk 1]
```diff
@@ -40,8 +40,8 @@
         }
     ],
     "wof:id":85686515,
-    "wof:lastmodified":1457144800,
-    "wof:name":"Distrito Federal",
+    "wof:lastmodified":1463766802,
+    "wof:name":"Ciudad de M\u00e9xico",
```
Comment (nvkelso) [inline data/856/865/15/85686515.geojson]: Should the old name be preserved in something like:

`wof:name`:`esp_x_variant` = `Distrito Federal`?

**Comment by thisisaaronland** [comment]
Yes, that seems wise. `_x_colloquial` maybe or even `_x_historical`. We can add whatever we want after `_x_` so long as it's documented :D

**Comment by thisisaaronland** [comment]
otherwise 👍

**Pull request merged**